test/OpenFeature.Tests/OpenFeatureHookTests.cs

```diff
@@ -394,7 +491,7 @@ public async Task Error_Hook_Should_Be_Executed_Even_If_Abnormal_Termination()
             hook1.BeforeAsync(Arg.Any<HookContext<bool>>(), null).Returns(EvaluationContext.Empty);
             hook2.BeforeAsync(Arg.Any<HookContext<bool>>(), null).Returns(EvaluationContext.Empty);
             featureProvider1.ResolveBooleanValueAsync(Arg.Any<string>(), Arg.Any<bool>(), Arg.Any<EvaluationContext>()).Throws(new Exception());
-            hook2.ErrorAsync(Arg.Any<HookContext<bool>>(), Arg.Any<Exception>(), null).Returns(new ValueTask());
+            hook2.ErrorAsync(Arg.Any<HookContext<bool>>(), Arg.Any<Exception>(), null).Throws(new Exception());

```

Comment:
The test description is for an `error` hook abnormally terminating, but it didn't have an error hook abnormally terminate. As a result the catch block in the error hook was untested.  Based on the ordering and test description I think this hook was supposed to throw.

Reply:
Yes I think you're right.